src/Symfony/Bundle/FrameworkBundle/Resources/config/form_debug.xml

```diff
@@ -26,7 +26,6 @@
         <service id="data_collector.form" class="Symfony\Component\Form\Extension\DataCollector\FormDataCollector" public="true">
             <tag name="data_collector" template="@WebProfiler/Collector/form.html.twig" id="form" priority="310" />
             <argument type="service" id="data_collector.form.extractor" />
-            <argument>false</argument>
```

Comment:
Actually this one should be done in lower branches too. See https://github.com/symfony/symfony/pull/23048